cl/recap/mergers.py

```diff
@@ -2132,6 +2133,42 @@ def process_case_query_report(
     return None
 
 
+def find_scotus_lower_court(lower_court_name: str) -> Court | None:
```

Comment:
Seems like this should be a general purpose utility, since it has no scotus-specific code, right? 

Also, did you check for other places courts-db is in use? Maybe we already have some good examples of it from the case law team?

Reply:
I think it's safe to assume that no bankruptcy gets appealed to SCOTUS. Bankruptcy cases are appealed to the circuit court first, and then to SCOTUS from there.